cpp/src/column/column.cu

```diff
@@ -283,6 +373,13 @@ column::column(column_view view, cudaStream_t stream,
   column{std::move(*experimental::type_dispatcher(view.type(),
                     create_column_from_view{view, stream, mr}))} {}
 
+// Allow strategy switching at runtime for easier benchmarking
+// TODO remove when done
+static concatenate_mode current_mode = concatenate_mode::UNOPTIMIZED;
```

Comment:
I know this doesn't matter since it's temporary, but this would be better as a function local static.

```
concatenate_mode& get_concatenate_mode(){
   static cocatenate_mode mode{cocatenate_mode::UNOPTIMIZED};
   return mode;
}
...
get_concatenate_mode() = concatenate_mode::BINARY_SEARCH;
```

https://isocpp.org/wiki/faq/ctors#static-init-order-on-first-use

https://en.wikibooks.org/wiki/More_C%2B%2B_Idioms/Construct_On_First_Use